frontend/io-webapp/src/session/Sessionbar.js

```diff
@@ -16,8 +16,8 @@ const Sessionbar = ({ state, dispatch }) => {
         }),
       })
         .then((_) => {
-          dispatch({type: "SET_SESSION_ID", payload: ""})
-          dispatch({type: "SET_STAGE_ACCOUNT"})
+          dispatch({ type: "SET_SESSION_ID", payload: "" });
+          dispatch({ type: "SET_STAGE_ACCOUNT" });
```

Comment:
Jakby w przyszłości robić jakiś refactor to dobrą praktyką jest albo stworzenie funkcji, które tworzą obiekty reduxowych akcji w stylu:
```
export const setSessionId = (sessionId) => ({
  type: "SET_SESSION_ID",
  payload: sessionId,
});
```
albo przynajmniej wrzucenie tych typów akcji w jakieś eksportowane stałe, tak, żeby uniknąć potencjalnych błędów spowodowanych takim wpisywaniem stringów z palca w wielu miejscach w kodzie.

Reply:
Zdecydowanie wygląda to lepiej, niż to co napisałem, ten refactor który robiłem tydzień temu był mocno inspirowany Reduxem, ale brakuje mi jeszcze trochę wiedzy i doświadczenia, więc wygląda na razie jak wygląda.